src/components/AddComments.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { postArticleVote } from "../utils/api";
+
+export default function AddComments({ article_id }) {
+  const [isPosted, setIsPost] = useState(false);
+  const [newComment, setNewComment] = useState({
+    username: "",
+    body: "",
+  });
+
+  useEffect(() => {
+    setIsPost(false);
+    if (newComment.username !== "") {
+      postArticleVote(article_id, newComment)
+        .then((res) => alert("Posted successfully. Please refresh the page."))
+        .catch((err) => {
+          alert("Please enter a correct username or valid comments. Please refresh the page.");
+        });
+      setIsPost(true);
+    }
+  }, [newComment, article_id]);
+
```

Comment:
Though you have posted the comment successfully to the database, it's not updating on the page (something which you're covering by asking the user to refresh the page)
However we can't be asking the user to refresh every time, what we need to do is use state to update the comments following a successful post request.
So we post the comment to the db, and then update the comments state so it includes the new posted comment
Can you please add this and then resubmit this PR